core/src/main/java/org/apache/accumulo/core/metadata/schema/ExternalCompactionMetadata.java

```diff
@@ -129,10 +129,11 @@ private static class GSonData {
   public String toJson() {
     GSonData jData = new GSonData();
 
-    jData.inputs = jobFiles.stream().map(StoredTabletFile::getMetaUpdateDelete).collect(toList());
-    jData.nextFiles =
-        nextFiles.stream().map(StoredTabletFile::getMetaUpdateDelete).collect(toList());
-    jData.tmp = compactTmpName.getMetaInsert();
+    jData.inputs = jobFiles.stream()
+        .map(tabletFile -> tabletFile.getMetaUpdateDelete().getMetaString()).collect(toList());
```

Comment:
This breaks when/if `getMetaString` changes to include the file range.

Reply:
So in this case to keep the changes smaller I was simply trying to do a 1 to 1 mapping on the update in this PR and the old version was also using getMetaUpdateDelete() which used to return a String so this is the same as before.

I can probably update this to just use the file reference since that's really what it needs but ultimately all of this will need to be updated regardless with the new range in a future PR so I'm not sure how worthwhile it is to make too many changes to this right now knowing a lot more will be changing in future PRs as well.